packages/sanity/src/core/comments/components/pte/comment-input/CommentInputProvider.tsx

```diff
@@ -60,124 +57,154 @@ export function CommentInputProvider(props: CommentInputProviderProps) {
 
   const editor = usePortableTextEditor()
 
-  const [activeCaretElement, setActiveCaretElement] = useState<HTMLElement | null>(null)
   const [mentionsMenuOpen, setMentionsMenuOpen] = useState<boolean>(false)
+  const [mentionsSearchTerm, setMentionsSearchTerm] = useState<string>('')
   const [selectionAtMentionInsert, setSelectionAtMentionInsert] = useState<EditorSelection>(null)
 
-  // todo
   const canSubmit = useMemo(() => {
     if (!value) return false
 
     return value?.some(
-      (block: FIXME) => (block?.children || [])?.some((c: FIXME) => c.text || c.userId),
+      (block) =>
+        isPortableTextTextBlock(block) &&
+        (block?.children || [])?.some((c) => (isPortableTextSpan(c) ? c.text : c.userId)),
     )
   }, [value])
 
   const hasChanges = useCommentHasChanged(value)
 
-  const focusLastBlock = useCallback(() => {
-    const block = PortableTextEditor.focusBlock(editor)
+  const focusEditor = useCallback(() => PortableTextEditor.focus(editor), [editor])
```

Comment:
Would it be possible to update this function to set the cursor at the end if there is already a value on focus? Currently, when you click "Edit Comment", the cursor is placed at the beginning of the message. But it would be nice if it was placed at the end!

Reply:
Yes, I'll look into it. I think it preferably should be an explicit function that gets called on edit - and not a general focus function here.

Reply:
@hermanwikner - see my two lastest commits (new)